assets/characters/adventurer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "adventurer",
+  "texture": "characters/adventurer.png",
+  "stats": {
+    "vitality": 30,
+    "hitPoints": 30,
+    "speed": 3,
+    "maxAP": 3,
+    "currentAP": 3,
+    "attack": {
+      "die": [
+        8
+      ],
+      "mod": 1
+    }
+  },
+  "behaviour": "player"
+}
```

Comment:
This is fine for now but we might want to structure things differently down the line. Effectively we're going to want a way to have shared asset packs with tile-sets, character drawings and maybe some boring generic mobs but we're also going to want to have some less generic stuff that's specific to a story. 

I'm imagining we'd have a campaign which would be a set of levels, characters, dialogue trees etc. etc. that make up a story. I suppose we can create resource packs that represent some common stuff that you might want to share between campaigns and then the campaign would library these in. 

I guess just keep this in mind before we get to married to this format as it will probably need to change down the line.